--- a/amprd.c	2025-05-08 13:32:32.854731778 -0300
+++ b/amprd.c	2025-05-08 13:33:27.531478763 -0300
@@ -65,7 +65,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <netinet/ip.h>
-#include <netinet/udp.h>
+#include <linux/udp.h>
 #include <linux/if_ether.h>
 #include <sys/ioctl.h>
 #include <arpa/inet.h>
@@ -84,6 +84,11 @@
 #include "callhome.h"
 #include "commons.h"
 
+#ifndef IPPORT_ROUTESERVER
+#define IPPORT_ROUTESERVER 520
+#endif
+
+
 #define MAXPAYLOAD 		(2048)
 
 char *passwd = NULL;
